app/__tests__/app.test.js

```diff
@@ -28,6 +28,55 @@ describe("GET /topics", () => {
     })
 })
 
+describe("GET /api", () => {
+    describe("-- functionality tests", () => {
+        test("200: responds with 200 status code and a JSON object with description, queries and exampleResponse keys with correct data types", () => {
+            return request(app)
+            .get("/api")
+            .expect(200)
+            .then(({body}) => {
+                const endPointsObj = body.endpoints
+                for(let key in endPointsObj) { 
+                    expect(endPointsObj[key]).toMatchObject({
+                        description: expect.any(String), 
+                        queries: expect.any(Object),
+                        exampleResponse: expect.any(Object)
+                    })
+                }
+                
+            })
+        })
+        test("200: responds with 200 status code and the queries key is a valid array, this array can be empty", () => {
+            return request(app)
+            .get("/api")
+            .expect(200)
+            .then(({body}) => {
+                const endPointsObj = body.endpoints
+                for(let key in endPointsObj) { 
+                    expect(Array.isArray(endPointsObj[key].queries)).toBe(true)
+                }                
+            })
+        })
```

Comment:
Im not sure why this is a separate test, this can and should be part of the first test.